src/styles/pages/_cpa.scss

```diff
@@ -0,0 +1,34 @@
+.giraffe {
+  .cpa {
+
+    #unknown_user {
+      background-color: $lighter-gray;
+      padding:27px;
+      .input-block {
+      
+        label {
+          font-size: 15px;
+        }
+
+        input {
+          width: 100%;
+          max-width:100%;
```

Comment:
I don't think `max-width` is needed if `width` is specified, but I could be wrong! Was this added to address a particular case?

Reply:
Good catch, removed!